src/routes/+layout.svelte

```diff
@@ -52,8 +57,13 @@
     height: auto;
     width: 100%;
   }
+  .gradient {
+    background-image: linear-gradient(#f0a9a8 75%, #9e9e9f00 100%);
+  }
   .logo-desktop {
     width: 100%;
+    z-index: 10;
+    padding-bottom: 30px;
   }
   .logo-mobile {
```

Comment:
logo-desktop & logo-mobile are exactly the same apart from the display prop, please cleanup